packages/xo-web/src/common/render-xo-item.js

```diff
@@ -229,20 +229,27 @@ Sr.defaultProps = {
 export const Vdi = decorate([
   connectStore(() => {
     const getObject = createGetObject()
+    const getSr = createGetObject((state, props) => {
+      const vdi = getObject(state, props, props.self)
+      return vdi && vdi.$SR
+    })
     // FIXME: props.self ugly workaround to get object as a self user
     return (state, props) => ({
       vdi: getObject(state, props, props.self),
+      sr: getSr(state, props, props.self),
```

Comment:
Self users shouldn't be able to see *any* SR name just because their VDIs are sitting on them. Let's remove the `props.self`.